scripts/tests/integration_tests.bats

```diff
@@ -0,0 +1,242 @@
+#!/usr/bin/env bats
+
+load "bats/bats-support/load.bash"
+load "bats/bats-assert/load.bash"
+
+setup_file() {
+  _example_run_command="CHAIN_NAME=GAIA TRANSFER_CHANNEL_NUMBER=0 bats gaia_tests.bats"
+  if [[ "$CHAIN_NAME" == "" ]]; then 
+    echo "CHAIN_NAME variable must be set before running integration tests (e.g. $_example_run_command)" >&2
+    return 1
+  fi
+
+  if [[ "$TRANSFER_CHANNEL_NUMBER" == "" ]]; then 
+    echo "TRANSFER_CHANNEL_NUMBER variable must be set before running integration tests (e.g. $_example_run_command)" >&2
+    return 1
+  fi
+
+  # set allows us to export all variables in account_vars
+  set -a
+  source scripts/vars.sh
+
+  HOST_CHAIN_ID=$(GET_VAR_VALUE  ${CHAIN_NAME}_CHAIN_ID)
+  HOST_DENOM=$(GET_VAR_VALUE     ${CHAIN_NAME}_DENOM)
+  HOST_IBC_DENOM=$(GET_VAR_VALUE IBC_${CHAIN_NAME}_CHANNEL_${TRANSFER_CHANNEL_NUMBER}_DENOM)
+  HOST_MAIN_CMD=$(GET_VAR_VALUE  ${CHAIN_NAME}_MAIN_CMD)
+
+  HOST_VAL_ADDRESS=$(${CHAIN_NAME}_ADDRESS)
+  HOST_RECEIVER_ADDRESS=$(GET_VAR_VALUE ${CHAIN_NAME}_RECEIVER_ADDRESS)
+
+  HOST_VAL="$(GET_VAR_VALUE ${CHAIN_NAME}_VAL_PREFIX)1"
+  STRIDE_VAL=${STRIDE_VAL_PREFIX}1
+
+  STRIDE_TRANFER_CHANNEL="channel-${TRANSFER_CHANNEL_NUMBER}"
+  HOST_TRANSFER_CHANNEL="channel-0"
+
+  TRANSFER_AMOUNT=500000
+  STAKE_AMOUNT=100000
+  REDEEM_AMOUNT=1000
+
+  GETBAL() {
+    head -n 1 | grep -o -E '[0-9]+' || "0"
+  }
+  GETSTAKE() {
+    tail -n 2 | head -n 1 | grep -o -E '[0-9]+' | head -n 1
+  }
+  # HELPER FUNCTIONS
+  DECADD () {
+    echo "scale=2; $1+$2" | bc
+  }
+  DECMUL () {
+    echo "scale=2; $1*$2" | bc
+  }
+  FLOOR () {
+    printf "%.0f\n" $1
+  }
+  CEIL () {
+    printf "%.0f\n" $(ADD $1 1)
+  }
+
+  set +a
+}
+
+##############################################################################################
+######                              HOW TO                                              ######
+##############################################################################################
+# Tests are written sequentially
+# Each test depends on the previous tests, and examines the chain state at a point in time
+
+# To add a new test, take an action then sleep for seconds / blocks / IBC_TX_WAIT_SECONDS
+#     / epochs
+# Reordering existing tests could break them
+
+##############################################################################################
+######                              SETUP TESTS                                         ######
+##############################################################################################
+# confirm host zone is registered
+@test "[INTEGRATION-BASIC-$CHAIN_NAME] host zones successfully registered" {
+  run $STRIDE_MAIN_CMD q stakeibc show-host-zone $HOST_CHAIN_ID
+  assert_line "  host_denom: $HOST_DENOM"
```

Comment:
Does it make sense to log `TRANSFER_CHANNEL_NUMBER`?

Reply:
by log, do you mean add it to the assert? If so I think that's a great idea!
I don't think it would make sense to log from the integration tests though

Reply:
Sorry - meant assert yes